doc/source/management/data_at_rest_encryption.rst

```diff
@@ -532,6 +532,10 @@ tables.
    |MariaDB| Documentation: ``innodb_encrypt_tables`` Option
       https://mariadb.com/kb/en/library/xtradbinnodb-server-system-variables/#innodb_encrypt_tables
 
+.. note:: 
+
+If the 5.7 version has encrypted MySQL schema tables due to ``--innodb-encrypt-tables`` ="ON" during bootstrap or if ``--innodb_encrypt-tables`` is present at the upgrade time, the create mysql tablespace is encrypted during upgrade.
```

Comment:
There is no "create mysql tablespace" - the jira ticket means that the mysql tablespace is created as encrypted.

It also should be `innodb-encrypt-tables`, or `innodb_encrypt_tables`, consistently, not mixed.

`innodb-encrpyt-tables` also doesn't exist anymore in 8.0. The first mention (5.7 bootstrap) is still true), but the second (8.0 upgrade) isn't anymore. @satya-bodapati According to upstream docs, the new `default_table_encryption` doesn't encrypt the mysql tablespace, which means now only the first option is possible?

Reply:
Yes, I wasn't sure how 8.0 would handle a 5.7 upgrade.